custom_components/simple_wyze_vac/vacuum.py

```diff
@@ -312,6 +312,12 @@ async def async_send_command(self, command, params=None, **kwargs):
         
         elif command in ["get_map", "get_last_map"]:
             await self.get_last_map()
+
+        elif command in ["set_map", "set_current_map"]:
+            if "map" in params:
```

Comment:
To catch when user doesn't include `map` as a parameter
```suggestion
            if params and "map" in params:
```